packages/lib/modules/pool/LbpDetail/LbpHeader/LbpHeaderTitleDescription.tsx

```diff
@@ -15,7 +15,7 @@ export function LbpHeaderTitleDescription() {
   const socialLinks = [
     {
       iconType: 'x',
-      href: `https://twitter.com/${lbpPool.x}`,
+      href: lbpPool.x ? `https://twitter.com/${lbpPool.x}` : undefined,
```

Comment:
@groninge01 

i heard from @agorer that we decided to make twitter handle optional so am suggesting we he icon on pool page if undefined